.travis.yml

```diff
@@ -7,12 +7,20 @@ os: osx
 osx_image: xcode6.4
 
 env:
+  matrix:
+    
+    - CONDA_PY=27
```

Comment:
@jerowe and @scopatz I tracked the problem of the missing packages down to a missing dependency when the `feedstock` was merged, probably due to a race condition of the staged-recipes PR with multiple recipes.

That should result in an error but latest changes in `conda-smithy` still renders the recipe with missing dependencies :scream:!!! In addition to that `conda-build` does not exit with an error when building and empty package... So we can only see the failure if we inspect the CI log.